plugins/amazonq/codewhisperer/jetbrains-community/build.gradle.kts

```diff
@@ -12,5 +12,7 @@ intellijToolkit {
 }
 
 dependencies {
-    compileOnly(project(":plugin-amazonq:shared:jetbrains-community"))
+    implementation(project(":plugin-amazonq:shared:jetbrains-community"))
+
+    compileOnly(project(":plugin-core:jetbrains-community"))
```

Comment:
Would this also be required for implementation for the common bits?

Reply:
we want to be able to compile but avoid pulling in the jars at runtime because of the plugin classloader issue when the same class has been loaded by another plugin